examples/MANN/config/config_joypad.toml

```diff
@@ -1,10 +1,18 @@
 base_vel_norm = 0.4
-ellipsoid_forward_axis = 1.0
-ellipsoid_side_axis = 0.9
-ellipsoid_backward_axis = 0.6
+# For vel-based MANN
+base_ang_vel_norm = 0.4
+ellipsoid_forward_axis = 3.0
+ellipsoid_side_axis = 2.0
+ellipsoid_backward_axis = 3.0
 ellipsoid_scaling_factor = 0.4
+# For original MANN
 max_facing_direction_angle_forward = 0.20
 max_facing_direction_angle_backward = 0.1
 max_facing_direction_angle_side_opposite_sign = 0.26
 max_facing_direction_angle_side_same_sign = 0.17
+# For vel-based MANN
+max_base_direction_angle_forward = 0.79
+max_base_direction_angle_backward = 0.1
+max_base_direction_angle_side_opposite_sign = 0.26
+max_base_direction_angle_side_same_sign = 0.17
```

Comment:
To simplify the tutorial duplicating them shouldn't be problematic.

Reply:
Done by separating VelMANN in examples. dcf45280